# Cancel robot's goal
bool cancelGoal